pages.da/common/grep.md

```diff
@@ -0,0 +1,36 @@
+# grep
+
+> Find mønstre i filer via regulære udtryk (regular expressions).
+> Mere information: <https://www.gnu.org/software/grep/manual/grep.html>.
+
+- Søg efter et mønster i en fil:
+
+`grep "{{søgemønster}}" {{sti/til/fil}}`
+
+- Søg efter en eksakt streng (deaktiverer regulære udtryk):
+
+`grep --fixed-strings "{{eksakt_streng}}" {{sti/til/fil}}`
+
+- Søg efter et mønster i alle filer, pånær binære, rekursivt i en mappe. Vis linjenumre der matcher til mønstret:
```

Comment:
I checked this with Google Translate shouldn't this be ` Search for a pattern in all files recursively in a directory, showing line numbers of matches, ignoring binary files:`'s Danish equivalent. 

Your reads `Search for a pattern in all files, including binaries, recursively in a folder. Show line numbers that match the pattern:`

Reply:
Hi @kbdharun, my translation reads correctly in Danish. The "pånær" negates subsequent word, i.e., "binære". In English that would be "except binary".

The only change from the same line in the English tldr is moving the "ignoring binary files" part into the sentence, rather than having it by the end, which works better in Danish.

Original sentence, from English grep.md:
Search for a pattern in all files recursively in a directory, showing line numbers of matches, ignoring binary files: